src/parser/paladin/holy/modules/checklist/Component.js

```diff
@@ -114,10 +114,19 @@ class HolyPaladinChecklist extends React.PureComponent {
           />
         </Rule>
         <Rule
-          name={<Trans>Don't tunnel the tanks</Trans>}
+          name={<Trans>Beacon Mastery</Trans>}
```

Comment:
Hey I think your new Requirements warrant a new Rule called something like "Keep your beacons active". For the Checklist a rule of thumb is to try to word checklist rules like rule of thumbs. This should be easier to understand and remember for users, and makes it easier for us to decide how to format the checklist. This also allows you to write a custom description to try to help players.

Beacon Mastery might also be slightly misleading for players who played in Legion, as Beacon Mastery was an important part of Beacon of the Lightbringer back then.

Reply:
Changed per request